tests/analysis_tests/tests/src/expected/SignatureHelp.res.txt

```diff
@@ -9,11 +9,12 @@ ContextPath Value[someFunc]
 Path someFunc
 argAtCursor: unlabelled<0>
 extracted params: 
-[int, ~two: string=?, ~three: unit => unit, ~four: someVariant, unit]
+[(
```

Comment:
Need to look into these changes: the loc of parameters has changed.

Reply:
Given this function type:
```
type t = (  x: int
         ^  ^
         f  t
```
The parser updates the location of the top-level node to start at location `f`.

So in the case of `function$< Tarrow>` the top-level node starts at `f` and the `Tarrow` node starts at `t`.
Now that `Tarrow` is the top-level node, it starts at `f`.

Neither is optimal, perhaps at some point one could consider a better representation of function types, where the locations of the overall type, and individual labels is straightforward. (Labels themselves are a little funny with optional parameters).

Reply:
CC @zth